### PR TITLE
V8/bug/ab4899 remember user filtering

### PR DESCRIPTION
**Pull request by nielslyngsoe**
Remember user filtering, by storing it in the URL parameters, like the other filtering options.

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/views/components/umb-mini-search/umbminisearch.component.js [hunk 1]
```diff
@@ -10,7 +10,8 @@
             bindings: {
                 model: "=",
                 onStartTyping: "&?",
```
Comment (warrenbuckley) [inline src/Umbraco.Web.UI.Client/src/views/components/umb-mini-search/umbminisearch.component.js]: Do we drop `on-start-typing` from this component/directive and get all implementations to use the `on-blur` approach @nielslyngsoe ?

Comment (nielslyngsoe) [reply src/Umbraco.Web.UI.Client/src/views/components/umb-mini-search/umbminisearch.component.js]: No, just for this specific usage of umb-mini-search. So still used in other places.

**Comment by warrenbuckley** [comment]
Code seems fine to me, but off to test now :)

**Comment by warrenbuckley** [comment]
All tested and works fine, once I was testing on the right branch 😃

**Pull request merged**